### PR TITLE
Fixing broken links

### PR DESCRIPTION
**Pull request by KCeh**
Various links were either formated or fixed.

### CODE REVIEW

File: Building quality apps/Naming.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ if (checkFileSize(image))
 * Activities and fragments are named with the appropriate suffix: `LoginActivity`, `RegisterFragment`.
 * The corresponding layouts have to be named with the appropriate prefix: `activity_login.xml`, `fragment_register.xml`.
 * All naming has to be simple, clear and mnemonic (short and meaningful).
-* The [Infinum Code Style](https://github.com/infinum/android-handbook-private/blob/master/files/InfinumCodeStyle.xml) has to be imported into the Android Studio and used in formatting code. If you want to find out how to import it, check out the chapter about [Development environment setup](/books/android/onboarding/development-environment-setup).
+* The [Infinum Code Style]() has to be imported into the Android Studio and used in formatting code. If you want to find out how to import it, check out the chapter about [Development environment setup](/books/android/onboarding/development-environment-setup).
```
Comment (KCeh) [inline Building quality apps/Naming.md:54]: I couldn't find link to Infinum code style for Android...

File: Useful tools & utilities/Logging.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Timber.w(...)
 Timber.wtf(...)
 ```
 
-For more information about methods, see the javadoc [here](http://jakewharton.github.io/timber).
+For more information about methods, see the javadoc [here](https://jakewharton.github.io/timber/docs/5.x/index.html).
```
Comment (KCeh) [inline Useful tools & utilities/Logging.md:28]: This is version specific link, but only such links will work

File: Building quality apps/Naming.md (hunk 1 above)
Comment (ffloreani) [reply Building quality apps/Naming.md:54]: Right here in the public handbook 😄  https://github.com/infinum/android-handbook/blob/master/files/InfinumCodeStyle.xml

File: Useful tools & utilities/Logging.md (hunk 2 above)
Comment (ffloreani) [reply Useful tools & utilities/Logging.md:28]: Confirmed 😢 but hey, better to have something than to not have a link at all

**Review by ffloreani — APPROVED** [review]
ne-vidim-problem

**Pull request merged**